extensions/reactive-oracle-client/runtime/pom.xml

```diff
@@ -15,6 +15,14 @@
     <name>Quarkus - Reactive Oracle Client - Runtime</name>
     <description>Connect to the Oracle database using the reactive pattern</description>
     <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-jdbc-oracle</artifactId>
```

Comment:
humm that seems wrong, no? it would pull in all transitive dependencies to actually start a JDBC pool

Reply:
Can you elaborate about the JDBC specific registrations?